app/main.py

```diff
@@ -1,3 +1,20 @@
+
 def cache(func):
-    # Write your code here
-    pass
+
+    d = {}
```

Comment:
`d` is a bad variable name (please check yourself before pushing by using the `checklist.md`)